src/main/java/com/jmethods/catatumbo/DefaultQueryResponse.java

```diff
@@ -96,4 +103,19 @@ public void setEndCursor(DatastoreCursor endCursor) {
     this.endCursor = endCursor;
   }
 
+  @Override
+  public QueryResultBatch.MoreResultsType getMoreResultsType() {
```

Comment:
Since Catatumbo is a framework that wraps the GCloud API, we do not want to expose any of the Objects from the underlying API. We need to create something within Catatumbo, that is equivalent to `QueryResultBatch.MoreResultsType` and expose that. Would you please make the necessary change and update the PR? Thanks!

Reply:
Updated!